templates/components/account/order-contents.html

```diff
@@ -21,11 +21,12 @@ <h5 class="account-listShipping-title">{{lang 'account.orders.details.ship_to_mu
             </div>
             <figure class="account-product-figure">
                 {{#if type '===' 'giftcertificate'}}
-                    <img src="{{cdn ../../theme_settings.default_image_gift_certificate}}" alt="GiftCertificate">
+                    <img src="{{cdn ../../theme_settings.default_image_gift_certificate}}" alt="GiftCertificate" title="GiftCertificate">
```

Comment:
`alt` and `title` should be readable by humans. So "Gift certificate" instead of "GiftCertificate". `title` text should ideally be different to `alt`.